src/types.ts

```diff
@@ -3243,6 +3260,25 @@ export enum HostEvent {
      * @hidden
      */
     UIPassthrough = 'UiPassthrough',
+    /**
+     * Triggers the table viz rerender with the updated data.
+     * Includes the following properties:
+     * @param - columnDataLite - an array of object containing data
+     * transformed from data picked from TableVizRendered event.
+     * For example, { columnDataLite: []}
+     * @example
+     * ```js
+     * searchEmbed.on(EmbedEvent.TableVizRendered, (payload) => {
+     *       console.log(payload);
+     *       const columnDataLite = payload.data.data.columnDataLite;
+     *       columnDataLite[0].dataValue[0]="new fob";
+     *       console.log('>>> new Data', columnDataLite);
+     *       searchEmbed.trigger(HostEvent.TransformTableVizData, columnDataLite);
```

Comment:
can you update docs for TransformTableVizData example

Reply:
Done